File: step01example/src/main/java/com/example/step01example/MainActivity.java
package com.example.step01example;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button sendBtn = findViewById(R.id.sendBtn);
        sendBtn.setOnClickListener(this);

        Button sendBtn2 = findViewById(R.id.sendBtn2);
        sendBtn2.setOnClickListener(view -> {
            TextView t = findViewById(R.id.textView);
            EditText e = findViewById(R.id.editText);
            t.setText(e.getText().toString());
        });
    }

    @Override
    public void onClick(View view){
        TextView t = findViewById(R.id.textView);
        EditText e = findViewById(R.id.editText);
        t.setText(e.getText().toString());
    }
}

File: step07sqlite2/src/main/java/com/example/step07sqlite2/MainActivity.kt
package com.example.step07sqlite2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.view.View.OnClickListener
import android.widget.Toast
import com.example.step07sqlite2.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() , OnClickListener { // extends AppCompatActivity

    // View Binding 객체를 저장할 필드
    lateinit var binding: ActivityMainBinding
    // var binding: ActivityMainBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // getLayoutInflater() 대신에 LayoutInflater 라고 작성하면 된다.
        binding = ActivityMainBinding.inflate(layoutInflater)

        // Layout 구성
        setContentView(binding.root)

        // Button 에 Listener 등록
        binding.sendBtn.setOnClickListener(this)
    }

    override fun onClick(p0: View?) {
        // 입력한 문자열을 읽어와서 .getText() 대신에 .text
        val msg = binding.inputMsg.text.toString()

        // Toast 에 띄우기
        Toast.makeText(this , msg , Toast.LENGTH_SHORT).show()
    }
}